### ISSUE TITLE
NTR: Coproporphyrin high or abnormal pattern in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8856]
New term request Coproporphyrin high or abnormal pattern in urine (segal_120228220845): Coproporphyrinuria, coproporphyria; tetracarboxylporphyrin

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000536
- label: Elevated urinary coproporphyrin level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: 
- def: The amount of a coproporphyrin in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8856
- synonyms:
